app/controllers/searches_controller.rb

```diff
@@ -140,7 +140,7 @@ def docs_search_klass
   end
 
   def redesign?
```

Comment:
For better code quality I suggest to move this method to `ApplicationController` and remove it from the controllers.